### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ArthurNRL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -370,6 +370,19 @@ docker).
 A previous version of this action did things with the wiki. This is not the case
 anymore.
 
+## .coverage file generated on a non-unix file system
```
Comment (ewjoachim) [inline README.md]:
```suggestion
## .coverage file generated on a Windows file system
```

File: README.md [hunk 2]
```diff
@@ -370,6 +370,19 @@ docker).
...
+
+If your project needs to be built and tested on a non-unix os adding
```
Comment (ewjoachim) [inline README.md]:
`````suggestion
If your project's coverage was built on Windows, you may get an error like:
```
CoverageWarning: Couldn't parse 'yourproject\__init__.py': No source for code: 'yourproject\__init__.py'. (couldnt-parse)
```
This is likely due to coverage being confused with the coverage being computed with `\` but read with `/`. You can most probably fix it with the following in your [coverage configuration](https://coverage.readthedocs.io/en/latest/config.html):
`````

File: README.md [hunk 3]
```diff
@@ -370,6 +370,19 @@ docker).
...
+If your project needs to be built and tested on a non-unix os adding 
+
+```
+[paths]
+source =
+    */project/module
+    *\project\module
+```
+
+to .coveragerc will help the action to find the covered files.
```
Comment (ewjoachim) [inline README.md]:
```suggestion
```

**Review by ewjoachim — COMMENTED** [review]
Thank you very much ! I suggested some improvements but we're on track :)

"non-unix" vs "windows": to the best on my knowledge, Windows is the odd one here. Using / for path separators is usual even outside of UNIX (though I'm not an expert of non-unix non-windows OSes)

**Comment by ArthurNRL** [comment]
@ewjoachim updated

**Comment by ewjoachim** [comment]
The PR points to your own fork. I f you want tu submit this to the repo, you need to edit or recreate the PR so that it points to the v3 on the upstream fork. I'll do this for you if you want.

**Comment by ewjoachim** [comment]
(You can close this PR)

**Pull request closed without merging**